packages/plugin-ext/src/main/browser/file-system-main.ts

```diff
@@ -14,13 +14,16 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { interfaces, injectable } from 'inversify';
+import { interfaces, injectable, inject } from 'inversify';
 import Uri from 'vscode-uri';
 import { Disposable, ResourceResolver, DisposableCollection } from '@theia/core';
 import { Resource } from '@theia/core/lib/common/resource';
 import URI from '@theia/core/lib/common/uri';
 import { MAIN_RPC_CONTEXT, FileSystemMain, FileSystemExt } from '../../common/plugin-api-rpc';
 import { RPCProtocol } from '../../common/rpc-protocol';
+import { FileSystemError } from '../../plugin/types-impl';
```

Comment:
we should not access plugin code from browser code

Reply:
ok, need to clean up that in other modules probably ?

Reply:
I thought we cleaned it up already. Do you see it imported somewhere else? maybe it slipped again, we really need linting rules for dependencies control between js modules :(

Reply:
![image](https://user-images.githubusercontent.com/436777/73335229-0885a200-426f-11ea-92e2-160dad07cdf2.png)

Reply:
oh, filed https://github.com/eclipse-theia/theia/issues/7015, let's tackle it separate